src/main/java/org/apache/commons/io/FileUtils.java

```diff
@@ -2393,9 +2393,17 @@ public static void moveFile(final File srcFile, final File destFile) throws IOEx
      * @since 2.9.0
```

Comment:
Either way this needs a @throws IllegalArgumentException here for the case where the srcFile exists but is a directory, unless we want that to be an IOException too. And the method's doc comment should be clearer that directories are not supported. Unless maybe we want to delegate that case to moveDirectory and not throw any exception at all?